js/src/vm/JSScript.h

```diff
@@ -1783,11 +1783,11 @@ class JSScript : public js::BaseScript {
   }
 
   size_t pcToOffset(const jsbytecode* pc) const {
-    MOZ_ASSERT(containsPC(pc));
-    return size_t(pc - code());
+    return immutableScriptData()->pcToOffset(pc);
   }
 
   jsbytecode* offsetToPC(size_t offset) const {
+    return immutableScriptData()->offsetToPC(offset);
     MOZ_ASSERT(offset < length());
     return code() + offset;
```

Comment:
We have some unreachable code here

Reply:
Ah, result of a bad merge a while ago, fixed!